extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -240,7 +240,8 @@ export class PublishDatabaseDialog {
 				connectionUri: await this.getConnectionUri(),
 				sqlCmdVariables: this.getSqlCmdVariablesForPublish(),
 				deploymentOptions: await this.getDeploymentOptions(),
-				profileUsed: this.profileUsed
+				profileUsed: this.profileUsed,
```

Comment:
Can this flag not just be a check for whether `publishProfileUri !== undefined`?

Reply:
Yes, that should suffice. Updated accordingly.